src/pages/dutch-pay/calculate.ts

```diff
@@ -0,0 +1,41 @@
+import { Member, Transfer } from './interface';
+
+interface RoundProps {
+  members: Member[];
+  transfers: Transfer[];
+}
+
+function round({ members, transfers }: RoundProps): [Member[], Transfer[]] {
+  const newMembers = members.map((member) => new Member(member));
+  newMembers.sort((a, b) => b.diff - a.diff);
+
+  const sender = newMembers[0];
+  const recipient = newMembers.find((member) => member.diff < 0);
+  if (!recipient) { return [newMembers, transfers]; }
+
+  const newTransfers = [...transfers, new Transfer({
+    id: transfers.length + 1,
+    from: sender.name,
+    to: recipient.name,
+    amount: sender.diff,
+  })];
+
+  recipient.setDiff(recipient.diff + sender.diff);
+  sender.setDiff(0);
+  return [newMembers, newTransfers];
+}
+
+export function calculate(inputMembers: Member[]) {
+  let transfers: Transfer[] = [];
+  const totalPaid = inputMembers.reduce((acc, member) => acc + member.paid, 0);
+  const average = Math.floor(totalPaid / inputMembers.length);
+  let members = inputMembers
+    .map((member) => new Member(member))
+    .map((member) => member.calculateDiff(average));
+
+  for (let i = 0; i < members.length; i += 1) {
+    [members, transfers] = round({ members, transfers });
+  }
+
+  return { members, transfers };
+}
```

Comment:
```suggestion
  return { members, transfers };
}

```
혹시 line feed가 자동으로 적용되지 않는가요? 린트 설정 해놨는데 흠..

Reply:
음 이게 제가 포맷터를 각 개발마다 다르게 설정하고 진행해야해서 아마 적용이 안된부분이 있을수도 있을 것 같네요ㅜ
린트에러 알림이 따로 안떠서 캐치를 못한 것 같습니다